assets/src/block-editor/plugins/amp-preview-item.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import { AMPFilledIcon } from '../../icons';
+import AmpPreviewMenuItem from '../components/preview-menu-item';
+
+export const name = 'amp-preview-menu-item';
+
+export const icon = (
+	<AMPFilledIcon viewBox="0 0 62 62" height={18} width={18} />
```

Comment:
The styling of the menu items is a bit different:

![image](https://github.com/user-attachments/assets/1bc17c87-f1f7-4237-82b5-3f81f3b2c3db)

![image](https://github.com/user-attachments/assets/6c7dd0d6-704c-4e6d-ab55-80b96736d53f)

Maybe this is contributing to the icon being cropped at the bottom.

Reply:
If I make the icon have the 24x24 pixel dimensions then it isn't cropped. But then it's probably too big.

Reply:
Interesting, With the latest build on this PR, it looks like

![image](https://github.com/user-attachments/assets/97b620f8-bcec-47a5-85c9-7b52a0b63ebe)

Reply:
Likewise @thelovekesh, The latest build doesn't have the cropping issue on any breakpoints.